acbuild/acbuild.go

```diff
@@ -239,6 +244,12 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 }
 
 func main() {
+	if path.Base(os.Args[0]) == "acbuild-script" {
+		os.Args[0] = path.Join(path.Dir(os.Args[0]), "acbuild")
+		args := os.Args[1:]
+		os.Args = []string{os.Args[0], "script"}
+		os.Args = append(os.Args, args...)
+	}
```

Comment:
Perhaps I'm missing something, but looks like this is related to cobra/pflags argument parsing (so cobra will execute `cmdScript`) than multicall.